tests/functional/tests/osd/test_journal_collocation.py

```diff
@@ -4,4 +4,4 @@ class TestOSD(object):
     def test_osds_are_all_collocated(self, node, Command):
         # TODO: figure out way to paramaterize node['vars']['devices'] for this test
         for device in node["vars"]["devices"]:
-            assert Command.check_output("sudo blkid -s PARTLABEL -o value %s2" % device) == "ceph journal"
+            assert Command.check_output("sudo blkid -s PARTLABEL -o value %s2" % device) == ("ceph journal" or "ceph block")
```

Comment:
``assert Command.check_output("sudo blkid -s PARTLABEL -o value %s2" % device) in ["ceph journal", "ceph block"]`` would be better I think